site/content/en/latest/concepts/client-traffic-policy.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "ClientTrafficPolicy"
+---
+## Before you Begin
+- [Gateway API Extensions](gateway-api-extensions.md)
+
+## Overview
+
+`ClientTrafficPolicy` is an extension to the Kubernetes Gateway API that controls how Envoy Gateway handles incoming traffic from clients. It can configure connection behavior, security settings, and protocol optimizations at the gateway level.
+
+It can act as a gatekeeper that manages how external clients connect to your services, helping you maintain security, performance, and stability.
+
+## Use Cases
+
+`ClientTrafficPolicy` is particularly useful in scenarios where you need to:
+
+- **Strengthen security:** Implement IP allow/deny lists and authenticate clients with mutual TLS
+- **Optimize performance:** Enable HTTP/3 and fine-tune connection settings
+- **Protect resources:** Limit connections and implement rate limiting to prevent overload
+- **Preserve client identity:** Maintain original client IP addresses when behind load balancers
+- **Monitor and Control:** Track connection metrics, monitor client behavior and control access patterns
+
+## `ClientTrafficPolicy` in Envoy Gateway
+
+`ClientTrafficPolicy` is part of the Envoy Gateway API suite, which extends the Kubernetes Gateway API with additional capabilities. It's implemented as a Custom Resource Definition (CRD) that you can use to configure how Envoy Gateway manages incoming client traffic.
+
+You can apply a `ClientTrafficPolicy` to a Gateway resource, and it will affect how that gateway handles client connections. The policy's effects are applied at the listener level, with some important distinctions:
+
+- For HTTP listeners: All HTTP listeners in a Gateway share a common connection counter
+- For HTTPS/TLS listeners: Each listener maintains its own separate connection counter
+
+This separation ensures that different types of traffic can be managed independently while still maintaining overall control over gateway resources.
+
+## Best Practices
```

Comment:
same as above

Reply:
re thinking, it may be best to create a new top level section called Best Practices so we can include specific things like https://github.com/envoyproxy/gateway/issues/5851 in there